pkgs/by-name/ku/kubetui/package.nix

```
@@ -0,0 +1,33 @@
+{
+  rustPlatform,
+  lib,
+  fetchFromGitHub,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "kubetui";
+  version = "1.5.3";
+
+  src = fetchFromGitHub {
+    owner = "sarub0b0";
+    repo = "kubetui";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-0K0h/MaQClJDqgF0qQO2INb+hpzxfSikAti+751MX/8=";
+  };
+
```

```suggestion
  buildInputs = lib.optionals stdenv.hostPlatform.isDarwin [
    darwin.apple_sdk.frameworks.CoreGraphics
    darwin.apple_sdk.frameworks.AppKit
  ];
```

(This is the full list)